package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "node-test": "mocha test/**/*-test.js",
+    "browser-test": "COVERAGE=true ember test",
```

Comment:
The 'integration' node test runs COVERAGE=true ember test within ember-cli-code-coverage, so this might be extra work.